package.json

```diff
@@ -31,7 +31,7 @@
     "redux": "^4.1.2",
     "stellar-sdk": "^9.1.0",
     "styled-components": "^5.3.3",
-    "trezor-connect": "^8.2.2",
+    "trezor-connect": "8.2.4-extended",
```

Comment:
the version info does not completely follow Semantic Versioning, so I think we have to specify a specific version.

Reply:
Do you know what is the difference between `8.2.4` and `8.2.4-extended`? I see that we have always used the base version in the past.
This is not a big deal but would help to keep things as simple as possible to avoid questions in the future.